### PR TITLE
added jenkins build script

### PR DESCRIPTION
**Pull request by brucehoff**

### CODE REVIEW

File: jenkins_build.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# customize each private build here
+user=$1
+org_sagebionetworks_stack_iam_id=$2
+org_sagebionetworks_stack_iam_key=$3
+org_sagebionetworks_stackEncryptionKey=$4
+
+stack=dev
+user_name=${stack}${user}
```
Comment (brucehoff) [inline jenkins_build.sh]: rds_user_name

File: jenkins_build.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+</org.sagebionetworks.id.generator.database.connection.url>\
+</properties></profile></profiles></settings>" > /var/lib/jenkins/${JOB_NAME}/.m2/settings.xml
+
+
+# start up rds container
+docker run --name ${rds_container_name} \
+-e MYSQL_ROOT_PASSWORD=default-pw \
+-e MYSQL_DATABASE=${user_name} \
+-e MYSQL_USER=${user_name} \
+-e MYSQL_PASSWORD=platform \
```
Comment (brucehoff) [inline jenkins_build.sh]: pass in mysql_password from host

File: jenkins_build.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+bash -c "mvn clean install \
+-Dorg.sagebionetworks.stackEncryptionKey=${org_sagebionetworks_stackEncryptionKey} \
+-Dorg.sagebionetworks.stack.iam.id=${org_sagebionetworks_stack_iam_id} \
+-Dorg.sagebionetworks.stack.iam.key=${org_sagebionetworks_stack_iam_key} \
+-Dorg.sagebionetworks.stack.instance=${user} \
+-Dorg.sagebionetworks.developer=${user} \
+-Dorg.sagebionetworks.stack=${stack} \
+-Dorg.sagebionetworks.repositoryservice.endpoint=http://localhost:8080/services-repository-develop-SNAPSHOT/repo/v1 \
+-Dorg.sagebionetworks.fileservice.endpoint=http://localhost:8080/services-repository-develop-SNAPSHOT/file/v1 \
+-Duser.home=/root"
```
Comment (brucehoff) [inline jenkins_build.sh]: may not need ll. 65-66.

File: jenkins_build.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+maven:3-jdk-7 \
+bash -c "mvn clean install \
+-Dorg.sagebionetworks.stackEncryptionKey=${org_sagebionetworks_stackEncryptionKey} \
+-Dorg.sagebionetworks.stack.iam.id=${org_sagebionetworks_stack_iam_id} \
+-Dorg.sagebionetworks.stack.iam.key=${org_sagebionetworks_stack_iam_key} \
+-Dorg.sagebionetworks.stack.instance=${user} \
+-Dorg.sagebionetworks.developer=${user} \
+-Dorg.sagebionetworks.stack=${stack} \
+-Dorg.sagebionetworks.repositoryservice.endpoint=http://localhost:8080/services-repository-develop-SNAPSHOT/repo/v1 \
+-Dorg.sagebionetworks.fileservice.endpoint=http://localhost:8080/services-repository-develop-SNAPSHOT/file/v1 \
```
Comment (brucehoff) [inline jenkins_build.sh]: Per John, can pass DB URL by -"D<name>"=<value>

File: jenkins_build.sh [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+--link ${rds_container_name}:${rds_container_name} \
+-v /var/lib/jenkins/${JOB_NAME}/.m2:/root/.m2 \
+-v /var/lib/jenkins/workspace/${JOB_NAME}:/repo \
+-v /etc/localtime:/etc/localtime:ro \
+-e MAVEN_OPTS="-Xms256m -Xmx2048m -XX:MaxPermSize=512m" \
+-w /repo \
+maven:3-jdk-7 \
+bash -c "mvn clean install \
+-Dorg.sagebionetworks.repository.database.connection.url=jdbc:mysql://${rds_container_name}/${rds_user_name} \
+-DJDBC_CONNECTION_STRING=jdbc:mysql://${rds_container_name}/${rds_user_name} \
```
Comment (john-hill) [inline jenkins_build.sh]: remove this one

**Pull request merged**